### ISSUE TITLE
Seems dailymotion update again, then clicktoplugin fails

### ISSUE THREAD

**Issue by Roy75117** [hoyois/clicktoplugin#167]
http://goo.gl/NTYSxp

As the dailymotion link above, seems that dailymotion update its html5 player.

And now, clicktoplugin fail to load mp4 player.

If possible, could you fix it? I really need clicktoplugin for dailymotion.

Thanks

**Comment by JohnBlackburne** [comment]
Works fine for me as HTML, i.e. using whatever player they supply. It takes a few seconds to load but then plays the Japanese TV show.

**Comment by Roy75117** [comment]
Hello seems not work for me.
You can see the snapshot below, it is dailymotion html5 player
http://imgur.com/s9LWgY7

Here is the html source code from my safari
http://goo.gl/qf9L6Y

**Comment by JohnBlackburne** [comment]
Yes, that’s what I’m seeing, a working HTML5 player. How is that “not work"? HTML5 is just mp4 and vice-versa, at leat on Safari. By poking around I could get a direct link to the video source (actually a playlist):

http://www.dailymotion.com/cdn/manifest/video/x42r0vm.m3u8?auth=1462824582-2688-eotqmf1x-e7da44fa773f212d80b26fe460696f62

And that just plays in Safari as an mp4.

**Comment by Roy75117** [comment]
I am just wondering is it possible to be the player in Youtube?

Like this, there are lots of function from right mouse click. i.e., send to airplay, save as a file, copy the url of the video,etc... 
http://imgur.com/Yi5VBQn
